relay-general/src/protocol/device_class.rs

```diff
@@ -0,0 +1,17 @@
+use crate::types::Annotated;
+use std::fmt;
+
+#[derive(Debug, FromValue, IntoValue, Empty, Clone, PartialEq)]
+pub struct DeviceClass(pub u64);
+
+impl DeviceClass {
+    pub const LOW: Self = Self(1);
+    pub const MEDIUM: Self = Self(2);
+    pub const HIGH: Self = Self(3);
+}
```

Comment:
Hey Joris, we plan to add additional device classes as higher performing devices get released in the market. Rather than store explicitly as `high medium and low`, we want to store them as number values (even though they'll be strings). This is so we have some ascending sequence of numbers and map those numbers to user facing values of `high medium and low` as we scale up. (ie one day we may have 4 -> low, 5 -> medium, 6 -> high)

Reply:
👍 Makes sense.